src/components/TeacherComponents/UnitsLearningThemeFilters/RadioTile.tsx

```diff
@@ -0,0 +1,117 @@
+import styled from "styled-components";
+
+import {
+  OakBox,
+  OakBoxProps,
+  OakFlex,
+  OakLabel,
+  OakLabelProps,
+} from "@/styles/oakThemeApp";
+
+// TODO: extract to components lib
+const RadioTileLabel = styled(OakLabel)<OakLabelProps>`
+  cursor: pointer;
+  display: flex;
+  gap: 8px;
+`;
+
+const HiddenRadioButtonInput = styled.input.attrs({
+  type: "radio",
+})`
+  position: absolute;
+  opacity: 0;
+  cursor: pointer;
+`;
+
+const RadioTileFocus = styled(OakBox)<OakBoxProps>`
+  box-shadow: ${`inset 0 0 0 0.15rem #ffe555`};
```

Comment:
Is there a reason to use a template for the box-shadow here?

Reply:
nope! it's because I had originally tried to get the colour hex from the token name but that fn isn't exported from components library 🫠